pgr-services/src/main/java/org/egov/pgr/service/MigrationService.java

```diff
@@ -54,6 +55,21 @@
         }
     };
 
```

Comment:
add the updated states which I have sent you

Reply:
have added the same one here